packages/scss/src/components/_container.scss

```diff
@@ -1,21 +1,21 @@
 .container {
 	margin: 0 auto;
 	padding: 0 2rem;
-	width: $xl - 40px;
+	width: _get($luTheme, "breakpoints.xl") - 40px;
 
-	@media (max-width: $xl) {
-		width: $lg - 40px;
+	@media (max-width: _get($luTheme, "breakpoints.xl")) {
```

Comment:
Lucca-ui mixins `lui_screens_larger_than()` and `lui_screens_smaller_than()` were kinda useful for developers always forgeting media queries syntax. Only problem was, it wasn't clear wether it was strict or not...